src/app/src/util/constants.js

```diff
@@ -715,3 +715,37 @@ export const facilitySidebarActions = {
     CLAIM_FACILITY: 'Claim this facility',
     VIEW_ON_OAR: 'View on the Open Apparel Registry',
 };
+
+export const EXTENDED_FIELD_TYPES = [
```

Comment:
The constant array of extended field types includes anticipated fields which aren't yet included in the API